API/MDBApi.js

```diff
@@ -0,0 +1,8 @@
+export const getWeatherFromApi = () => {
```

Comment:
Actually, you service could have 3 functions : 
- One that fetches the data
- One that converts those raw data to a more convenient format that you will use in your components.
- One that combines the two others.

So, you'll be able to test the 2 last functions (using mocks for the API call ;) )